build_tools/python/e2e_test_artifacts/cmake_generator/iree_rule_generator.py

```diff
@@ -44,8 +44,10 @@ def build_model_import_rule(
 
     model = imported_model.model
     if model.source_type == common_definitions.ModelSourceType.EXPORTED_LINALG_MLIR:
-      if source_model_rule.file_path != str(output_file_path):
-        raise ValueError("Separate path for Linalg model isn't supported.")
+      if pathlib.Path(source_model_rule.file_path) != output_file_path:
```

Comment:
Any reason not to use `pathlib.PurePath` here? (The file path can contain CMake variable name, so they are not real path on the filesystem.)

Reply:
Sure, that works too.